Makefile

```diff
@@ -0,0 +1,29 @@
+# import the metadata given in __init__.py
+include chebpy/__init__.py
+PROJECT_VERSION := ${__version__}
+
+SHELL := /bin/bash
+
+.PHONY: packages help test tag
+
+
+.DEFAULT: help
+
+help:
+	@echo "make test"
+		@echo "       Run all tests"
+	@echo "make tag"
+		@echo "       Make a tag on Github."
+	@echo "make server"
+		@echo "       Start the Flask server."
+
+
+test: venv
+	$(VENV)/pip install -r ./tests/requirements.txt
+	$(VENV)/py.test -cov=chebpy -vv ./tests
+
+tag: test
+	git tag -a ${PROJECT_VERSION} -m "new tag"
+	git push --tag
```

Comment:
Personally not a fan of `git push` inside a Makefile, especially when the target `tag` only suggests tagging. Besides, it seems that `--tags` is [not recommended](https://stackoverflow.com/a/5195913). Is this for GitHub Actions auto-tagging or for the developers?